### ISSUE TITLE
Unable to change input in beforeItemAdded

### ISSUE THREAD

**Issue by Stichoza** [bootstrap-tagsinput/bootstrap-tagsinput#250]
Unable to change entered value in `beforeItemAdded` event listener.

``` js
$('#my-tags').on('beforeItemAdded', function(event) {
    console.log("Received " + event.item);
    event.item = "bar";
    console.log("Adding " + event.item);
}).on('itemAdded', function(event) {
    console.log("Added " + event.item);
});
```

So when I enter `foo` and press enter, I see following in console:

```
Received foo
Adding bar
Added foo
```

**Comment by neverfox** [comment]
This is fixed by 2f1c902 but it's not in a release, so you won't get it if you're using Bower. I'm going to submit an issue about a new release.

**Comment by Sentence** [comment]
@neverfox Thanks a lot, helped me a lot, you can check [my solution here](https://github.com/bootstrap-tagsinput/bootstrap-tagsinput/issues/230#issuecomment-234932385).
